src/main.rs

```
@@ -0,0 +1,11 @@
+fn main() {
+    let x: Vec<u8> = vec![];
+    let y = x.len() == 0;
```

clippy

unused variable

(warning)